UKHO.SalesCatalogueFileShareServicesMock.API/UKHO.SalesCatalogueFileShareServicesMock.API/Controllers/FileShareServiceController.cs

```diff
@@ -96,7 +97,7 @@ public ActionResult DownloadFile(string batchId, string fileName)
             }
             if (!string.IsNullOrEmpty(fileName))
             {
-                bytes = fileShareService.GetFileData(configuration["HOME"], batchId, fileName);
+                bytes = fileShareService.GetFileData(configuration["HOME"], Sanitise(batchId), fileName);
```

Comment:
Since we're (rightly) sanitising the BatchId, should we also consider the same for the filename? The filename is not allowed to include any path characters. It would be harder to provide an allowed list of characters, but we could exclude `\`, `/`, and `:`

Reply:
Also just realised that this is a mock, so possibly not worth doing!